blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,267 @@
+
+## Improved debuggability of Kubernetes deployments
+
+Sourcegraph deployments using docker have had instrumentation available to admins for better monitoring and alerting for several releases. With Sourcegraph 3.14, the Kubernetes cluster deployment now has the same functionality available via the admin page -> Instrumentation or directly at sourcegraph.example.com/-/debug. From there you can trigger pprof profiles, request traces, and more for each server in a Sourcegraph cluster.
+
+## Monitoring and alerting added to the Sourcegraph instrumentation
+
+TODO: Screenshot/video?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+TODO:Intro sentence here. The frontend dashboard now shows in detail how search is behaving overall and at a glance. Additonally, admins can now be proactively notified when critical parts of Sourcegraph are not working as expected.
+
+Alerts are in place for when:
+
+* Hard search errors (both timeouts and general errors) are high.
+* Partial search timeouts are high.
+* Search 90th and 99th percentile request duration is high.
+* Users are being shown an abnormally large amount of search alert user suggestions and no results.
+* The internal indexed and unindexed search services are returning bad responses.
+* Gitserver may be under heavy load due to many concurrent command executions or under-provisioning.
+
+## Estimate required resources for your Sourcegraph instance
+
+TODO: screenshot
+
+The new [resource estimator](https://docs.sourcegraph.com/admin/install/resource_estimator) helps admins of new and growing Sourcegraph instances determine the right resources and deployment strategy for their custom scenario. CPU and memory needs are calculated for each service based on the number of repositories, users, large monorepos, and engagement rate of your users.
+
+## Repository management now can exclude GitHub forks/archives 
+
+Some organizations do not want to include any forks or archived repositories in their Sourcegraph instance. Admins can now exclude GitHub forks and/or archived repositories from the set of repositories being mirrored in Sourcegraph. The GitHub repository management can be updated with `"exclude": [{"forks": true}]` or `"exclude": [{"archived": true}]` to utilize this feature.
+
+## LSIF improvements to help advertise?
+
+Making the setup process better, links to browsing on open source projects. TBD - waiting on results from this week.
+
+## Aggregated search data added to Sourcegraph pings
```

Comment:
@dadlerj @ebrodymoore @attfarhan - please make sure this sufficiently describes the ping additions. I used the same format from the last blog post, but let me know if you think we should change or expand a bit more on why we are adding these pings.

Reply:
We also collect aggregated counts of total search users (daily, weekly, monthly). 

@rvantonder: search latency is mentioned here, in case there's something you want added/changed.

Reply:
Check it, reads fine to me.

Reply:
All of it looks good to me, adding the links to the pings and our process covers the bases. Thanks @christinaforney 😊